### ISSUE TITLE
(3.067) error after rename class in hierarchy view

### ISSUE THREAD

**Issue by dalehenrich** [ericwinger/Jade#465]
Renaming the class RwAdoptMethodErrorNotification to RwAdoptMissingMethodErrorNotification
![image](https://user-images.githubusercontent.com/623951/56917122-89f59d00-6a6f-11e9-9a95-7287fdaa29e3.png)
After refreshing the project window, it is clear that the rename had completed and that the references were properly replace, AND the post successful operation commit had been performed:
![image](https://user-images.githubusercontent.com/623951/56917524-8f071c00-6a70-11e9-96f1-1a7e54935f75.png)

Here is the Dolphin stack for the error:
```
************************** Dolphin Virtual Machine Dump Report ***************************

11:10:02 AM, 4/29/2019: Not found: #(a RowanMethodService(RwAdoptMethodErrorNotification>>_errorMessage))

*----> VM Context <----*
Process: {1D160210:size 421 words, suspended frame 1D160E15, priority 6, callbacks 1
last failure 2:nil, FP control 80017, thread 00000000}
Active Method: SessionManager>>logError:
IP: 0216C379 (9)
SP: 1D1609A8
BP: 1D16097C (460)
ActiveFrame: {1D160980: cf 1D160961, sp 1D160998, bp 1D16097C, ip 5, JadeSessionManager(SessionManager)>>logError:}
	receiver: a JadeSessionManager
	arg[0]: a NotFoundError


New Method: VMLibrary>>dump:path:stackDepth:walkbackDepth:
Message Selector: dump:path:stackDepth:walkbackDepth:

*----> Stack Back Trace <----*
{1D160980: cf 1D160961, sp 1D160998, bp 1D16097C, ip 5, JadeSessionManager(SessionManager)>>logError:}
	receiver: a JadeSessionManager
	arg[0]: a NotFoundError

{1D160960: cf 1D16093D, sp 1D160974, bp 1D160958, ip 39, JadeSessionManager>>logError:}
	receiver: a JadeSessionManager
	arg[0]: a NotFoundError
	stack temp[0]: '11:10:02 AM 4/29/2019'
	env temp[0]: a FileStream

{1D16093C: cf 1D16091D, sp 1D160950, bp 1D160938, ip 3, JadeSessionManager(SessionManager)>>unhandledException:}
	receiver: a JadeSessionManager
	arg[0]: a NotFoundError

{1D16091C: cf 1D1608FD, sp 1D160930, bp 1D160918, ip 3, JadeSessionManager(SessionManager)>>onUnhandledError:}
	receiver: a JadeSessionManager
	arg[0]: a NotFoundError

{1D1608FC: cf 1D1608E1, sp 1D160910, bp 1D1608FC, ip 5, NotFoundError(Error)>>defaultAction}
	receiver: a NotFoundError

{1D1608E0: cf 1D1608AD, sp 1D1608F4, bp 1D1608C8, ip 55, NotFoundError(Exception)>>_propagateFrom:}
	receiver: a NotFoundError
	arg[0]: a ExceptionHandler
	stack temp[0]: nil
	stack temp[1]: a ExceptionHandler
	stack temp[2]: nil
	stack temp[3]: a Process('Main' base 1D160000 [ACTIVE] in SessionManager>>logError: sp=00000000 ip=4 list=02610010)
	stack temp[4]: nil

{1D1608AC: cf 1D16088D, sp 1D1608C0, bp 1D1608A8, ip 6, NotFoundError(Exception)>>_propagate}
	receiver: a NotFoundError
	stack temp[0]: nil

{1D16088C: cf 1D160871, sp 1D1608A0, bp 1D16088C, ip 13, NotFoundError(Exception)>>signal}
	receiver: a NotFoundError

{1D160870: cf 1D16084D, sp 1D160884, bp 1D160868, ip 8, NotFoundError(Exception)>>signal:with:}
	receiver: a NotFoundError
	arg[0]: nil
	arg[1]: a Array

{1D16084C: cf 1D16082D, sp 1D160860, bp 1D160848, ip 5, NotFoundError(Exception)>>signalWith:}
	receiver: a NotFoundError
	arg[0]: a Array

{1D16082C: cf 1D16080D, sp 1D160840, bp 1D160828, ip 8, ListView(Object)>>errorNotFound:}
	receiver: a ListView
	arg[0]: a Array

{1D16080C: cf 1D1607ED, sp 1D160820, bp 1D160808, ip 12, [] in ListView(ListControlView)>>selections:}
	receiver: a ListView

{1D1607EC: cf 1D1607C1, sp 1D160800, bp 1D1607DC, ip 38, ListView>>selections:ifAbsent:}
	receiver: a ListView
	arg[0]: a Array
	arg[1]: [] @ 10 in ListControlView>>selections:
	stack temp[0]: a OrderedCollection
	stack temp[1]: a OrderedCollection

{1D1607C0: cf 1D1607A1, sp 1D1607D4, bp 1D1607BC, ip 14, ListView(ListControlView)>>selections:}
	receiver: a ListView
	arg[0]: a Array

{1D1607A0: cf 1D160781, sp 1D1607B4, bp 1D16079C, ip 17, [] in ListPresenter(SelectableItemsPresenter)>>selections:}
	receiver: a ListPresenter

{1D160780: cf 1D160755, sp 1D160794, bp 1D160770, ip 38, ListView>>selections:ifAbsent:}
	receiver: a ListView
	arg[0]: a Array
	arg[1]: [] @ 11 in SelectableItemsPresenter>>selections:
	stack temp[0]: a OrderedCollection
	stack temp[1]: a OrderedCollection

{1D160754: cf 1D160735, sp 1D160768, bp 1D160750, ip 19, ListPresenter(SelectableItemsPresenter)>>selections:}
	receiver: a ListPresenter
	arg[0]: a Array

{1D160734: cf 1D160705, sp 1D160748, bp 1D160720, ip 111, JadeiteProjectsBrowserPresenter>>refreshClassListOrHierarchy}
	receiver: a JadeiteProjectsBrowserPresenter
	stack temp[0]: nil
	stack temp[1]: nil
	stack temp[2]: nil
	stack temp[3]: a Array
	stack temp[4]: a Array

{1D160704: cf 1D1606D5, sp 1D160718, bp 1D1606F0, ip 29, JadeiteProjectsBrowserPresenter>>basicRenameClass:to:}
	receiver: a JadeiteProjectsBrowserPresenter
	arg[0]: a RowanClassService
	arg[1]: 'RwAdoptMissingMethodErrorNotification'
	stack temp[0]: a Array
	stack temp[1]: 'RwAdoptMethodErrorNotification'
	stack temp[2]: nil

{1D1606D4: cf 1D1606B1, sp 1D1606E8, bp 1D1606CC, ip 31, JadeiteProjectsBrowserPresenter>>renameClass}
	receiver: a JadeiteProjectsBrowserPresenter
	stack temp[0]: a RowanClassService
	stack temp[1]: 'RwAdoptMissingMethodErrorNotification'

{1D1606B0: cf 1D160691, sp 1D1606C4, bp 1D1606AC, ip 4, Symbol>>forwardTo:}
	receiver: renameClass
	arg[0]: a JadeiteProjectsBrowserPresenter

{1D160690: cf 1D160671, sp 1D1606A4, bp 1D16068C, ip 4, CommandDescription>>performAgainst:}
	receiver: a CommandDescription
	arg[0]: a JadeiteProjectsBrowserPresenter

{1D160670: cf 1D16064D, sp 1D160684, bp 1D160670, ip 14, [] in Command>>value}
	receiver: a Command

{1D16064C: cf 1D16062D, sp 1D160660, bp 1D160648, ip 18, BlockClosure>>ifCurtailed:}
	receiver: [] @ 0 in nil
	arg[0]: [] @ 17 in Command>>value

{1D16062C: cf 1D160609, sp 1D160640, bp 1D160624, ip 3, BlockClosure>>ensure:}
	receiver: [] @ 11 in Command>>value
	arg[0]: [] @ 17 in Command>>value
	stack temp[0]: nil

{1D160608: cf 1D1605ED, sp 1D16061C, bp 1D160608, ip 22, Command>>value}
	receiver: a Command

{1D1605EC: cf 1D1605CD, sp 1D160600, bp 1D1605E8, ip 30, ShellView>>performCommand:}
	receiver: a ShellView
	arg[0]: a Command

{1D1605CC: cf 1D1605AD, sp 1D1605E0, bp 1D1605C8, ip 4, JadeiteProjectsSystemBrowser(Shell)>>performCommand:}
	receiver: a JadeiteProjectsSystemBrowser
	arg[0]: a Command

{1D1605AC: cf 1D160585, sp 1D1605C0, bp 1D1605A0, ip 10, CommandQuery>>perform}
	receiver: a CommandQuery
	stack temp[0]: a Command
	stack temp[1]: nil
	stack temp[2]: a JadeiteProjectsSystemBrowser

{1D160584: cf 1D160561, sp 1D160598, bp 1D16057C, ip 8, DelegatingCommandPolicy(CommandPolicy)>>route:}
	receiver: a DelegatingCommandPolicy
	arg[0]: a CommandDescription
	stack temp[0]: a CommandQuery

{1D160560: cf 1D160539, sp 1D160574, bp 1D16055C, ip 15, [] in TreeView(View)>>onCommand:}
	receiver: a TreeView

{1D160538: cf 1D160519, sp 1D16054C, bp 1D160534, ip 18, BlockClosure>>ifCurtailed:}
	receiver: [] @ 0 in nil
	arg[0]: [] @ 18 in Cursor>>showWhile:

{1D160518: cf 1D1604F5, sp 1D16052C, bp 1D160510, ip 3, BlockClosure>>ensure:}
	receiver: [] @ 11 in View>>onCommand:
	arg[0]: [] @ 18 in Cursor>>showWhile:
	stack temp[0]: nil

{1D1604F4: cf 1D1604CD, sp 1D160508, bp 1D1604E8, ip 27, Cursor>>showWhile:}
	receiver: a Cursor
	arg[0]: [] @ 11 in View>>onCommand:
	stack temp[0]: nil
	stack temp[1]: a ExternalHandle(0001001F)

{1D1604CC: cf 1D1604A9, sp 1D1604E0, bp 1D1604C4, ip 17, TreeView(View)>>onCommand:}
	receiver: a TreeView
	arg[0]: a CommandDescription
	stack temp[0]: nil

{1D1604A8: cf 1D160475, sp 1D1604BC, bp 1D160490, ip 19, TreeView(View)>>wmCommand:wParam:lParam:}
	receiver: a TreeView
	arg[0]: 273
	arg[1]: 12349
	arg[2]: 0
	stack temp[0]: 12349
	stack temp[1]: a CommandDescription
	stack temp[2]: nil

{1D160474: cf 1D160445, sp 1D160488, bp 1D160460, ip 23, TreeView(View)>>dispatchMessage:wParam:lParam:}
	receiver: a TreeView
	arg[0]: 273
	arg[1]: 12349
	arg[2]: 0
	stack temp[0]: nil
	stack temp[1]: wmCommand:wParam:lParam:

{1D160444: cf 1D160411, sp 1D160458, bp 1D160434, ip 69, [] in InputState>>wndProc:message:wParam:lParam:cookie:}
	receiver: nil

{1D160410: cf 1D1603E9, sp 1D160424, bp 1D16040C, ip 18, BlockClosure>>ifCurtailed:}
	receiver: [] @ 0 in nil
	arg[0]: [] @ 79 in InputState>>wndProc:message:wParam:lParam:cookie:

{1D1603E8: cf 1D1603AD, sp 1D160404, bp 1D1603D0, ip 84, InputState>>wndProc:message:wParam:lParam:cookie:}
	receiver: a InputState
	arg[0]: 197792
	arg[1]: 273
	arg[2]: 12349
	arg[3]: 0
	arg[4]: 851284
	stack temp[0]: a TreeView

{1D1603AC: cf 1D16038D, sp 1D1603C8, bp 1D1603A8, ip 28, InputState>>pumpMessage:}
	receiver: a InputState
	arg[0]: a MSG

{1D16038C: cf 1D160361, sp 1D1603A0, bp 1D16037C, ip 18, InputState>>loopWhile:}
	receiver: a InputState
	arg[0]: [] @ 9 in InputState>>mainLoop
	stack temp[0]: a MSG
	stack temp[1]: true
	stack temp[2]: nil

{1D160360: cf 1D160345, sp 1D160374, bp 1D160360, ip 15, InputState>>mainLoop}
	receiver: a InputState

{1D160344: cf 1D160329, sp 1D160358, bp 1D160344, ip 14, [] in InputState>>forkMain}
	receiver: a InputState

{1D160328: cf 1D16030D, sp 1D16033C, bp 1D160328, ip 13, ExceptionHandler(ExceptionHandlerAbstract)>>markAndTry}
	receiver: a ExceptionHandler

{1D16030C: cf 1D1602E5, sp 1D160320, bp 1D160308, ip 22, [] in ExceptionHandler(ExceptionHandlerAbstract)>>try:}
	receiver: a ExceptionHandler

{1D1602E4: cf 1D1602C5, sp 1D1602F8, bp 1D1602E0, ip 18, BlockClosure>>ifCurtailed:}
	receiver: [] @ 0 in nil
	arg[0]: [] @ 39 in ExceptionHandlerAbstract>>try:

{1D1602C4: cf 1D1602A1, sp 1D1602D8, bp 1D1602BC, ip 3, BlockClosure>>ensure:}
	receiver: [] @ 17 in ExceptionHandlerAbstract>>try:
	arg[0]: [] @ 39 in ExceptionHandlerAbstract>>try:
	stack temp[0]: nil

{1D1602A0: cf 1D160279, sp 1D1602B4, bp 1D160294, ip 44, ExceptionHandler(ExceptionHandlerAbstract)>>try:}
	receiver: a ExceptionHandler
	arg[0]: [] @ 10 in InputState>>forkMain
	stack temp[0]: nil
	stack temp[1]: a Process('Main' base 1D160000 [ACTIVE] in SessionManager>>logError: sp=00000000 ip=4 list=02610010)
	env temp[0]: nil

{1D160278: cf 1D160255, sp 1D16028C, bp 1D160270, ip 7, BlockClosure>>on:do:}
	receiver: [] @ 10 in InputState>>forkMain
	arg[0]: ProcessTermination
	arg[1]: [] @ 13 in BlockClosure>>newProcess

{1D160254: cf 00000001, sp 1D160268, bp 1D160254, ip 17, [] in BlockClosure>>newProcess}
	receiver: [] @ 10 in InputState>>forkMain

<Bottom of stack>

***** End of dump *****
```

**Comment by ericwinger** [comment]
The reproduction case is: 
- Select a class with methods in the hierarchy view of a project browser
- Select several methods and ``Browse Selected Methods``
- With the method list open, rename the class. 
- get walkback shown above

**Comment by ericwinger** [comment]
Fixed in 3.0.69 https://github.com/GemTalk/Jadeite/commit/ba67da5d9da9720e35dc6324d2f1e2fa495ea436

**Comment by LisaAlmarode** [comment]
It doesn't walkback on rename, in 3.0.75.

However the method list browser that was left open does not get updated, and shows the old class name.

When I tried to right-click-pop up a menu on one of the methods in that list, I get a walkback.

```
a MessageNotUnderstood occurred (error 2010), a UndefinedObject does not understand  #'thisClass'

[1] MessageNotUnderstood >> defaultAction @2 line 3
[2] MessageNotUnderstood (AbstractException) >> _signalWith: @5 line 25
[3] MessageNotUnderstood (AbstractException) >> signal @2 line 47
[4] UndefinedObject (Object) >> doesNotUnderstand: @9 line 10
[5] UndefinedObject (Object) >> _doesNotUnderstand:args:envId:reason: @7 line 12
[6] RowanClassService >> theClass @14 line 6
[7] RowanClassService >> packageIsDirty @2 line 4
[8] RowanClassService >> arePackageAndProjectClean @2 line 3
[9] RowanClassService >> servicePerform:withArguments: @2 line 3
[10] [] in  ExecBlock1 (JadeServer) >> updateFromSton: @9 line 11
[11] OrderedCollection (Collection) >> do: @5 line 10
[12] JadeServer64bit32 (JadeServer) >> updateFromSton: @8 line 6
[13] GsNMethod class >> _gsReturnToC @1 line 1
===============================================
[1] MessageNotUnderstood >> defaultAction @2 line 3
	receiver oop(157895681) == a MessageNotUnderstood occurred (error 2010), a UndefinedObject does not understand  #'thisClass'

instance variables:
	-receiver oop(20) == nil
	-gsResumable oop(268) == true
	-gsNumber oop(16082) == 2010
	-currGsHandler oop(20) == nil
	-gsStack oop(20) == nil
	-gsReason oop(20) == nil
	-gsTrappable oop(268) == true
	-tag oop(20) == nil
	-messageText oop(157894401) == 'a MessageNotUnderstood occurred (error 2010), a UndefinedObject does not understand  #''thisClass'''
	-gsArgs oop(157895425) == anArray( )
	-selector oop(877057) == #'thisClass'
	-envId oop(2) == 0
	-gsDetails oop(20) == nil

temporary variables and parameters:
	result oop(20) == nil
----------------------------------------------------
[2] MessageNotUnderstood (AbstractException) >> _signalWith: @5 line 25
	receiver oop(157895681) == a MessageNotUnderstood occurred (error 2010), a UndefinedObject does not understand  #'thisClass'

instance variables:
	-receiver oop(20) == nil
	-gsResumable oop(268) == true
	-gsNumber oop(16082) == 2010
	-currGsHandler oop(20) == nil
	-gsStack oop(20) == nil
	-gsReason oop(20) == nil
	-gsDetails oop(20) == nil
	-gsTrappable oop(268) == true
	-messageText oop(157894401) == 'a MessageNotUnderstood occurred (error 2010), a UndefinedObject does not understand  #''thisClass'''
	-gsArgs oop(157895425) == anArray( )
	-selector oop(877057) == #'thisClass'
	-envId oop(2) == 0
	-tag oop(20) == nil

temporary variables and parameters:
	inCextensionArg oop(20) == nil
	res oop(20) == nil
----------------------------------------------------
[3] MessageNotUnderstood (AbstractException) >> signal @2 line 47
	receiver oop(157895681) == a MessageNotUnderstood occurred (error 2010), a UndefinedObject does not understand  #'thisClass'

instance variables:
	-envId oop(2) == 0
	-gsDetails oop(20) == nil
	-gsNumber oop(16082) == 2010
	-currGsHandler oop(20) == nil
	-gsStack oop(20) == nil
	-gsReason oop(20) == nil
	-gsTrappable oop(268) == true
	-tag oop(20) == nil
	-messageText oop(157894401) == 'a MessageNotUnderstood occurred (error 2010), a UndefinedObject does not understand  #''thisClass'''
	-gsArgs oop(157895425) == anArray( )
	-selector oop(877057) == #'thisClass'
	-receiver oop(20) == nil
	-gsResumable oop(268) == true

temporary variables and parameters:
----------------------------------------------------
[4] UndefinedObject (Object) >> doesNotUnderstand: @9 line 10
	receiver oop(20) == nil

instance variables:

temporary variables and parameters:
	aMessageDescriptor oop(157631233) == anArray( #'thisClass', anArray( ))
	args oop(157895425) == anArray( )
	ex oop(157895681) == a MessageNotUnderstood occurred (error 2010), a UndefinedObject does not understand  #'thisClass'
	sel oop(877057) == #'thisClass'
----------------------------------------------------
[5] UndefinedObject (Object) >> _doesNotUnderstand:args:envId:reason: @7 line 12
	receiver oop(20) == nil

instance variables:

temporary variables and parameters:
	aSymbol oop(877057) == #'thisClass'
	anArray oop(157895425) == anArray( )
	aSmallInt oop(2) == 0
	dnuKind oop(2) == 0
----------------------------------------------------
[6] RowanClassService >> theClass @14 line 6
	receiver oop(157830401) == aRowanClassService:Yellow

instance variables:
	-wasRemoved oop(12) == false
	-definition oop(20) == nil
	-command oop(35952641) == #'setIsTestCaseCommand'
	-commandArgs oop(157830145) == anArray( )
	-updateType oop(20) == nil
	-organizer oop(157829889) == aClassOrganizer
	-name oop(157829633) == 'Yellow'
	-comment oop(20) == nil
	-instVarNames oop(20) == nil
	-classVarNames oop(20) == nil
	-classInstVarNames oop(20) == nil
	-superclassName oop(20) == nil
	-subclassType oop(20) == nil
	-poolDictionaryNames oop(20) == nil
	-classType oop(20) == nil
	-meta oop(12) == false
	-isExtension oop(12) == false
	-version oop(20) == nil
	-versions oop(20) == nil
	-oop oop(20) == nil
	-updates oop(20) == nil
	-filters oop(157829377) == anArray( )
	-filterType oop(20) == nil
	-methods oop(157829121) == anArray( )
	-selectedPackageServices oop(157828865) == anArray( )
	-packageName oop(20) == nil
	-definedPackageName oop(20) == nil
	-selectedMethods oop(157828609) == anArray( )
	-projectName oop(20) == nil
	-hierarchyServices oop(20) == nil
	-variables oop(20) == nil
	-categories oop(20) == nil
	-isTestCase oop(12) == false
	-expand oop(20) == nil
	-visibleTests oop(157828353) == anArray( )
	-isNewClass oop(12) == false
	-updateAfterCommand oop(20) == nil
	-isInSymbolList oop(12) == false
	-dictionaryName oop(20) == nil
	-template oop(20) == nil

temporary variables and parameters:
	theClass oop(20) == nil
----------------------------------------------------
[7] RowanClassService >> packageIsDirty @2 line 4
	receiver oop(157830401) == aRowanClassService:Yellow

instance variables:
	-wasRemoved oop(12) == false
	-definition oop(20) == nil
	-command oop(35952641) == #'setIsTestCaseCommand'
	-commandArgs oop(157830145) == anArray( )
	-updateType oop(20) == nil
	-organizer oop(157829889) == aClassOrganizer
	-name oop(157829633) == 'Yellow'
	-comment oop(20) == nil
	-instVarNames oop(20) == nil
	-classVarNames oop(20) == nil
	-classInstVarNames oop(20) == nil
	-superclassName oop(20) == nil
	-subclassType oop(20) == nil
	-poolDictionaryNames oop(20) == nil
	-classType oop(20) == nil
	-meta oop(12) == false
	-isExtension oop(12) == false
	-version oop(20) == nil
	-versions oop(20) == nil
	-oop oop(20) == nil
	-updates oop(20) == nil
	-filters oop(157829377) == anArray( )
	-filterType oop(20) == nil
	-methods oop(157829121) == anArray( )
	-selectedPackageServices oop(157828865) == anArray( )
	-packageName oop(20) == nil
	-definedPackageName oop(20) == nil
	-selectedMethods oop(157828609) == anArray( )
	-projectName oop(20) == nil
	-hierarchyServices oop(20) == nil
	-variables oop(20) == nil
	-categories oop(20) == nil
	-isTestCase oop(12) == false
	-expand oop(20) == nil
	-visibleTests oop(157828353) == anArray( )
	-isNewClass oop(12) == false
	-updateAfterCommand oop(20) == nil
	-isInSymbolList oop(12) == false
	-dictionaryName oop(20) == nil
	-template oop(20) == nil

temporary variables and parameters:
	behavior oop(20) == nil
----------------------------------------------------
[8] RowanClassService >> arePackageAndProjectClean @2 line 3
	receiver oop(157830401) == aRowanClassService:Yellow

instance variables:
	-dictionaryName oop(20) == nil
	-oop oop(20) == nil
	-command oop(35952641) == #'setIsTestCaseCommand'
	-commandArgs oop(157830145) == anArray( )
	-updateType oop(20) == nil
	-organizer oop(157829889) == aClassOrganizer
	-name oop(157829633) == 'Yellow'
	-comment oop(20) == nil
	-instVarNames oop(20) == nil
	-classVarNames oop(20) == nil
	-classInstVarNames oop(20) == nil
	-superclassName oop(20) == nil
	-subclassType oop(20) == nil
	-poolDictionaryNames oop(20) == nil
	-classType oop(20) == nil
	-meta oop(12) == false
	-isExtension oop(12) == false
	-version oop(20) == nil
	-versions oop(20) == nil
	-updates oop(20) == nil
	-template oop(20) == nil
	-filters oop(157829377) == anArray( )
	-filterType oop(20) == nil
	-methods oop(157829121) == anArray( )
	-selectedPackageServices oop(157828865) == anArray( )
	-packageName oop(20) == nil
	-definedPackageName oop(20) == nil
	-selectedMethods oop(157828609) == anArray( )
	-projectName oop(20) == nil
	-hierarchyServices oop(20) == nil
	-variables oop(20) == nil
	-categories oop(20) == nil
	-isTestCase oop(12) == false
	-expand oop(20) == nil
	-visibleTests oop(157828353) == anArray( )
	-isNewClass oop(12) == false
	-updateAfterCommand oop(20) == nil
	-isInSymbolList oop(12) == false
	-wasRemoved oop(12) == false
	-definition oop(20) == nil

temporary variables and parameters:
----------------------------------------------------
[9] RowanClassService >> servicePerform:withArguments: @2 line 3
	receiver oop(157830401) == aRowanClassService:Yellow

instance variables:
	-wasRemoved oop(12) == false
	-definition oop(20) == nil
	-command oop(35952641) == #'setIsTestCaseCommand'
	-commandArgs oop(157830145) == anArray( )
	-updateType oop(20) == nil
	-organizer oop(157829889) == aClassOrganizer
	-name oop(157829633) == 'Yellow'
	-comment oop(20) == nil
	-instVarNames oop(20) == nil
	-classVarNames oop(20) == nil
	-classInstVarNames oop(20) == nil
	-superclassName oop(20) == nil
	-subclassType oop(20) == nil
	-poolDictionaryNames oop(20) == nil
	-classType oop(20) == nil
	-meta oop(12) == false
	-isExtension oop(12) == false
	-version oop(20) == nil
	-versions oop(20) == nil
	-oop oop(20) == nil
	-template oop(20) == nil
	-updates oop(20) == nil
	-filterType oop(20) == nil
	-methods oop(157829121) == anArray( )
	-selectedPackageServices oop(157828865) == anArray( )
	-packageName oop(20) == nil
	-definedPackageName oop(20) == nil
	-selectedMethods oop(157828609) == anArray( )
	-projectName oop(20) == nil
	-hierarchyServices oop(20) == nil
	-variables oop(20) == nil
	-categories oop(20) == nil
	-isTestCase oop(12) == false
	-expand oop(20) == nil
	-visibleTests oop(157828353) == anArray( )
	-isNewClass oop(12) == false
	-updateAfterCommand oop(20) == nil
	-isInSymbolList oop(12) == false
	-dictionaryName oop(20) == nil
	-filters oop(157829377) == anArray( )

temporary variables and parameters:
	collection oop(157830145) == anArray( )
	symbol oop(35952641) == #'setIsTestCaseCommand'
	wasClean oop(20) == nil
----------------------------------------------------
[10] [] in  ExecBlock1 (JadeServer) >> updateFromSton: @9 line 11
	receiver oop(157723649) == anExecBlock1

instance variables:

temporary variables and parameters:
	organizer oop(157829889) == aClassOrganizer
	service oop(157830401) == aRowanClassService:Yellow
----------------------------------------------------
[11] OrderedCollection (Collection) >> do: @5 line 10
	receiver oop(157818113) == anOrderedCollection( aRowanLoggingService, aRowanClassService:Yellow)

instance variables:

temporary variables and parameters:
	aBlock oop(157723649) == anExecBlock1
	i oop(18) == 2
----------------------------------------------------
[12] JadeServer64bit32 (JadeServer) >> updateFromSton: @8 line 6
	receiver oop(157875457) == aJadeServer64bit32

instance variables:
	-environment oop(2) == 0
	-classList oop(20) == nil
	-readStream oop(20) == nil
	-writeStream oop(157637121) == aWriteStream
	-selectedClass oop(20) == nil
	-methodFilterType oop(20) == nil
	-classOrganizers oop(157636865) == anArray( nil, nil, nil, nil)
	-selections oop(20) == nil
	-methodCommandResult oop(20) == nil
	-methodFilters oop(20) == nil

temporary variables and parameters:
	organizer oop(157829889) == aClassOrganizer
	resultString oop(20) == nil
	stonString oop(157896961) == 'OrderedCollection[RowanLoggingService{#command:#logReceivedServices,#commandArgs:[],#id:0,#groupId:12071,#services:OrderedCollection[RowanClassService{#name:''Yellow'',#command:#setIsTestCaseCommand,...
	services oop(157818113) == anOrderedCollection( aRowanLoggingService, aRowanClassService:Yellow)
----------------------------------------------------
[13] GsNMethod class >> _gsReturnToC @1 line 1
	receiver oop(144897) == GsNMethod

instance variables:

temporary variables and parameters:
```

**Comment by ericwinger** [comment]
This was a bit tricky to get right but I believe it's fixed  in 3.0.76 with the last work in progress commit. 
The difficulty arose from the fact that class service equality is based on name so simply renaming the service objects after the server operation would leave the lists & tree views in an unstable state.
